File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/MemoSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class MemoSearch extends AbstractEntity {
	String memoName;
	String memoDescription;
	@Temporal(TemporalType.DATE)
	Date memoDate;
}

File: scholar-web/src/main/webapp/app/model/staff/event/MemoSearch.js
Ext.define('scholar.model.staff.event.MemoSearch', {
	extend: 'Ext.data.Model',
	fields : [ {
		name : 'id',
		type : 'int'
	}, {
		name : 'memoName',
		type : 'string'
	}, {
		name : 'memoDescription',
		type : 'string'
	}, {
		name : 'memoDate',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ],
	
});